### ISSUE TITLE
Error while creating a project - the app stays on the 'create' page even though the project is created.

### ISSUE THREAD

**Issue by srish** [unstructuredstudio/zubhub#481]
I am trying to create a test project with a you tube video URL via "https://zubhub.unstructured.studio/projects/create." For some reason, the project gets created, but once that is done I am not taken to the https://zubhub.unstructured.studio/profile page. Instead, the app stays here https://zubhub.unstructured.studio/projects/create. And, I see a 500 error with
API POST request https://api.zubhub.unstructured.studio/api/projects/create/ in the console.

**Comment by souad988** [comment]
Hi @srish  
i created a new project using the same link https://zubhub.unstructured.studio/projects/create as a video link, but the project was created successfully and i was redirected to the profile page

**Comment by srish** [comment]
@souad988 Ahha, thanks for testing it. I made a mistake in the description, I meant YouTube video link and I remember it was happening when I was intentionally leaving errors in some other fields too.

**Comment by tuxology** [comment]
I just tested it. We do get a 500 error, but the project gets created

**Comment by darechan** [comment]
Hi!
Even I tried to recreate this issue using the same link but I did not get the error? Can you please elaborate more on what you did to get there?

![image](https://user-images.githubusercontent.com/93364240/219852854-c4537c41-5b7f-4a3b-b01e-43db07782660.png)
![image](https://user-images.githubusercontent.com/93364240/219852861-7aac3701-7c96-4d80-9515-e9bdac1871f5.png)

**Comment by srish** [comment]
@darechan Hi! Maybe try a couple of different options such as inputting a link to a YouTube video that is private or shows an error.

**Comment by ahn-nath** [comment]
@srish 
Hello, I am an Outreachy participant and I see this issue has not been assigned or requested before. I want to get it assigned to me, if possible. Thanks.

**Comment by tuxology** [comment]
@ahn-nath Can you verify if this is a valid issue as per @srish 's comment above? If it is, ping us back here and we will assign this to you

**Comment by ahn-nath** [comment]
@tuxology Yes, I will keep you posted.

**Comment by ahn-nath** [comment]
@tuxology, @srish 

I have tested the creation of a project four times with different variations:
- Leave random input field blank to get errors, resolve them and then submit the videos
- Setting to “Only Me” and “Everyone”. 

None of those have caused and error. I am redirected to the profile page every time. 
![zubhub2](https://user-images.githubusercontent.com/67247850/224226585-ef8b7d6c-032f-49ba-9627-ba59ccd8e45d.png)
![zubhub](https://user-images.githubusercontent.com/67247850/224226595-c572cbad-0217-4ebc-a79f-aa2da5c525af.png)

Maybe the way to generate the bug is more specific than that, or maybe, it was indirectly solved with a newer change or update to the repository.

**Comment by ahn-nath** [comment]
@tuxology 
Hello!

In my comments above I explained that after testing the project, I had no success verifying that the problem was still there. All of my attempts indicated the functionality is working as expected. If @srish can verify that the issue still persists, or recommends different ways of validating it (I attempted all possible ways I could imagine, but would be happy to try more), then I can take it. 

Let me know your thoughts.
